### DISCUSSION TITLE
Rinkeby的資料量超出預期

### DISCUSSION THREAD

**Discussion by frswim** [1101-GS4719-C/Python-Community-GS4719-C#450] [Q&A]
近期在研究區塊鏈，我想使用Rinkeby Network在上面建立區塊，利用web3和Rinkeby Network進行操作。
而為了要使用Rinkeby Network，我下載GETH v1.10.1的Windows版本。
安裝完成後利用以下指令進行與Rinkeby Network同步:
geth --rinkeby --syncmode "full"
然後我有查到相關資料提到Rinkeby Network的chaindata大概落在6gb，但是我載了20多gb他依舊再進行同步。
想請問有相關知識的大神是否有解決方法。
另外也有遇到 Snapshot extension registration failed以及Ethereum peer removal failed的問題，但我上網查找看起來沒什麼大礙。
![image](https://user-images.githubusercontent.com/99896928/157455698-dad3b9d6-176e-4442-bb27-8087ca74157d.png)

**Comment by SamTU0222** [comment]
我也想了解區塊鍊的知識
